apps/electric_migrations/lib/electric_migrations/postgres/generation.ex

```diff
@@ -20,20 +23,11 @@ defmodule Electric.Databases.Postgres.Generation do
     end
   end
 
-  def postgres_for_migrations_w_strings(migrations) do
-    originals_and_names =
-      for migration <- migrations do
-        %{name: migration["name"], original_body: migration["original_body"]}
-      end
-
-    postgres_for_migrations(originals_and_names)
-  end
-
   defp before_and_after_ast(migrations) do
-    with {:ok, after_ast, after_warnings} <- Parse.sql_ast_from_migrations(migrations),
+    with {:ok, after_ast, after_warnings} <- Parse.sql_ast_from_migrations(migrations, "public"),
```

Comment:
Sorry, I don't understand where a macro would fit in here. Could you elaborate please? I'm treating `Parse` as an internal module anyway, since the proper functions are exported via `ElectricMigrations.Postgres` and `ElectricMigrations.Sqlite` modules.

Reply:
Done